server/app/mutations/host_nodes/update.rb

```diff
@@ -4,19 +4,81 @@ module HostNodes
   class Update < Mutations::Command
 
     include Common
+    include Logging
 
     required do
       model :host_node
       array :labels, nils: true
     end
 
+    optional do
+      string :availability, in: ['active', 'drain']
```

Comment:
Wasn't there also some third state, which would just stop new instances from being scheduled to the node, but keep existing instances running?